scaffolds/cpp/Makefile

```diff
@@ -0,0 +1,32 @@
+cxxtestgen := cxxtestgen
+
+src = src
+src-files := \
+  $(src)/module.cpp
+
+header-files := \
+  $(src)/module.h
+
+test := test
+test-file := $(test)/tests.h
+
+build = build
+test-runner-src := $(build)/test-runner.cpp
+test-runner-exe := $(build)/test-runner
+
+.PHONY: all clean test
+
+all: test
```

Comment:
where is the binary?

Reply:
Which binary? There _is_ a rule in the makefile to build `$(test-runner-exe)`, if that is what you are asking.

Reply:
yep, I was asking that. why isn't it in the all target then?

Reply:
Ok, I changed the `all` target. You're right, it makes more sense.
- `make` just builds the test executable, without running the tests
- `make test` runs the tests